CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.0.6 (October 21st, 2022)
```

Comment:
nice try! :D

Reply:
I was merely saving you a menial task ;)  

Some providers demand a release notes version bump with PRs. I'll remove in this case.

Reply:
nah, the notes are good, the fixed date of today is what caught me :D

In this case I will probably indeed release today, so youre good
I use (unreleased) as date when I'm gathering change notes without a concrete release date